engineering/documents/README.tmpl.md

```diff
@@ -0,0 +1,53 @@
+```
+logo if any exists
+```
+
+```
+row of badgesyou might consider necessary.
+Examples include:
```

Comment:
Do you think it might be useful to include some actual examples of .md syntax with the badges, so people can just update it, instead of opening some other project and copying from there?

Reply:
Sure thing.

Reply:
Good idea! I added a couple of examples taken from go-git